client/src/components/admin/tables/ProjectsTable.tsx

```diff
@@ -73,8 +89,7 @@ const ProjectsTable = () => {
                 sortFunc = (a, b) => (a.seen - b.seen) * asc;
                 break;
             case ProjectSortField.Updated:
-                sortFunc = (a, b) =>
-                    (a.last_activity - b.last_activity) * asc;
+                sortFunc = (a, b) => (a.last_activity - b.last_activity) * asc;
                 break;
         }
         setProjects(unsortedProjects.sort(sortFunc));
```

Comment:
Can you add a sort field for `ProjectSortField.Flagged`? It should just sort all flagged projects to the top -- just add a case for it. Right now it does nothing and feels wack.

Reply:
for now can I just use the same sort that "seen" uses? That one seems to have the most overlap, ill make it acc work later.

Reply:
Yes but only if you create a new issue for this